contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1857,6 +1857,24 @@ class tsqlBuilder : public tsqlCommonMutator
 	void exitFunction_call(TSqlParser::Function_callContext *ctx) override
 	{
 		is_function = true;
+		if (ctx->NEXT())
```

Comment:
`exitFunction_call` is called not only for CREATE TABLE but also in other cases. Let's add more test cases other than CREATE TABLE case.